### PR TITLE
added input and image component

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs

## Screenshot:
input-and-image

## Deploy Preview Live:
https://deploy-preview-3--compi-ui.netlify.app/

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+.input-validator{
+    width: 100%;
+    position: relative;
+    border-radius: 0.5em;
+    border: 2px solid none;
+    padding: 0.5em 1em;
+}
+
+.input-success{
+    /* width: 35%; */
```
Comment (xii-vi) [inline components/input/input.css:29]: remove the unused property

**Review by xii-vi — APPROVED** [review]
One tiny suggestion, the rest is fine to merge.

File: components/input/input.css (hunk 1 above)
Comment (bishnoimukesh) [reply components/input/input.css]: removed.

**Review by sharmayatendra — COMMENTED** [review]
everything at place. you can merge

**Comment by bishnoimukesh** [comment]
suggested changes are made

**Pull request merged**